ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -3,18 +3,10 @@ base_images:
     name: commitchecker
     namespace: ci
     tag: "4.15"
-  ocp_4.18_base-rhel9:
-    name: "4.19"
-    namespace: ocp
-    tag: base-rhel9
   ocp_4.19_base-rhel9:
     name: "4.19"
     namespace: ocp
     tag: base-rhel9
-  ocp_builder_rhel-9-golang-1.22-openshift-4.18:
-    name: builder
-    namespace: ocp
-    tag: rhel-9-golang-1.22-openshift-4.18
```

Comment:
See: registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18
We are using this one:
https://github.com/openshift/operator-framework-operator-controller/blob/main/openshift/registry.Dockerfile#L1

So, I do not think that we can remove it yet

/hold

Reply:
See the PR: https://github.com/openshift/operator-framework-operator-controller/pull/235

then I think we can move with this one.

Reply:
We are likely fine, since all this file does is a round-about mechanism to get to back to the specific version. Order here shouldn't matter much.

Reply:
/unhold